### PR TITLE
Abaqus simplify USRinit + version 2022

### PR DESCRIPTION
**Pull request by hdolfen**
**Description** The USRInit does not check the increment number anymore to exit the DLOAD loop, because testing whether the loop has restarted is a stricter criterion, obsoleting the check of KINC. This makes the code also more robust against cases where the initial value of KINC is unexpected (like when "initial=NO" is not set for a dynamic step). 

Now also a warning is printed when "initial=NO" misses, as this setting is typically the desired behavior for an FSI simulation. This has been mentioned in the documentation too.

The documentation on how to set up a case has been clarified a bit.

Abaqus version 2022 is implemented.

**Users' experience affected?** Not significantly. Version 2022 became available. Now a warning is printed when a dynamic step does not contain "initial=NO" in the input file. Code slightly more robust.

**Tests** Unittests works. Relevant examples (containing Abaqus) have been evaluated using `evaluate_examples.py`.

**Related issues** #176 is partially solved by clarifying the required steps in the documentation. I don't close the issue yet because it also specifies the option of making things more automatic, which is not done in this pull request.

**Checklist**
- [x] Style guidelines
- [x] Self-review of code performed
- [x] Code has been commented (particularly in hard-to-understand areas)
- [x] Documentation was updated correspondingly
- [x] New and existing unit tests pass locally with changes

### CODE REVIEW

**Comment by nicolasdlss** [comment]
Is it necessary to have a new `setup_abaqus` folder (with the input file) or can it be copied from a previous version? (Like for example in Fluent)

File: coupling_components/solver_wrappers/solver_wrappers.md [hunk 1]
```diff
@@ -16,12 +16,12 @@ The `solve_solution_step` method is called in each coupling iteration by the cou
 
 There are currently two solver wrappers for computational fluid dynamics (CFD) packages:
 
--   [ANSYS Fluent](fluent/fluent.md) (2019R1, 2019R2, 2019R3, 2020R1)
+-   [ANSYS Fluent](fluent/fluent.md) (2019R1, 2019R2, 2019R3, 2020R1, 2021R1, 2021R2)
 -   [OpenFOAM](openfoam/openfoam.md) (4.1)
```
Comment (nicolasdlss) [inline coupling_components/solver_wrappers/solver_wrappers.md:20]: Can you also add 8, for OpenFOAM.

**Comment by hdolfen** [comment]
> Is it necessary to have a new `setup_abaqus` folder (with the input file) for the tests or can it be copied from a previous version? (Like for example in Fluent)

Yes, thanks for the suggestion, I will do this. It will simplify things and save some space.

File: coupling_components/solver_wrappers/abaqus/abaqus.py [hunk 2]
```diff
@@ -130,7 +130,6 @@ def initialize(self):
                         line = line.replace('|surfaces|', str(len(self.mp_in)))
                         line = line.replace('|surfaceIDs|', '\'' + '\', \''.join(self.mp_in) + '\'')
                         line = line.replace('|cpus|', str(self.cores))
-                        line = line.replace('|increment|', str(int(self.static)))
```
[the remark this answers is not in the document]
Comment (hdolfen) [reply coupling_components/solver_wrappers/abaqus/abaqus.py:133]: That is not anymore correct, it is now used in the `write_start_and_restart_inp` too check whether the input file contains the keyword *static or *dynamic correctly. But maybe we can remove this check as it is a bit strange to define a parameter just for a check?

**Pull request merged**